.circleci/config.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           command: |
             # Cleanup ansible environment on "remote" host. Order is important.
             sudo apt-get remove python-urllib3 python-setuptools
-            sudo apt-get update && sudo apt-get install -y python-dev python-pip
+            sudo apt-get update && sudo apt-get install -y python-dev python-pip virtualenv
```

Comment:
If I read the CI error correctly, this package name needs to be [python-virtualenv](https://packages.ubuntu.com/trusty/python-virtualenv).

https://circleci.com/docs/2.0/configuration-reference/#machine says that machine executor is Ubuntu 14.04.

Error at https://circleci.com/gh/aeternity/infrastructure/5448 is:
> E: Unable to locate package virtualenv

Reply:
I disabled the test for now, see PR description.